optuna/samplers/_base.py

```diff
@@ -212,6 +213,8 @@ def _process_constraints_after_trial(
     constraints = None
     try:
         con = constraints_func(trial)
+        if any(isnan(c) for c in con):
```

Comment:
Referring to other parts of Optuna, how about using `np.isnan`?

Reply:
Fixed.